hankel/hankel.py

```diff
@@ -298,18 +295,33 @@ def G(cls, f, h, k=None, *args, **kwargs):
 
         Returns
         -------
-        The value of G.
+        The value of G, the amplitude of the final term in the series' sum.
         """
         if k is None:
             return np.sqrt(2 * h / np.pi) * f(np.pi * np.pi / h)
         return np.sqrt(np.pi / (2 * h)) * f(np.pi * np.pi / h / k)
 
+    @classmethod
+    def G(cls, f, h, k=None, *args, **kwargs):
```

Comment:
Why is it still called `G` here and not `final_term_amplitude`?

Reply:
Good catch. Fixed it up.